week-4/implement/credit-card-vslidator.test.js

```diff
@@ -0,0 +1,30 @@
+// creditCard.test.js
+const creditCard = require('./credit-card-vslidator');
+
+describe('creditCard function', () => {
```

Comment:
Nice tests

Reply:
yes It works good I learned how test is worked mostly tests are passed but I think 2 test in password validator and 2 in card validator are failed I think the logic of the code has an error that's why some tests looks failed

Reply:
Which tests failed?